### PR TITLE
scipy >=1.7.1 < 1.10.0

### PR DESCRIPTION
**Pull request by weizmannk**
Scipy==1.10,  does not work  with bilby

### CODE REVIEW

**Comment by mcoughlin** [comment]
@weizmannk Can we setup ranges rather than pinning?

**Comment by weizmannk** [comment]
@mcoughlin  Yes , I will do it,

basically I think the result will be the same because look like scipy will try to install the last , 
the most recent version  agree by the code .

File: setup.py [hunk 1]
```diff
@@ -77,7 +77,7 @@ def readfile(filename):
     "bilby_pipe==1.0.5",
     "numpy>=1.9",
     "matplotlib>=2.0",
-    "scipy>=1.7.1",
+    "scipy==1.9.3",
```
Comment (weizmannk) [inline setup.py:80]:
```suggestion
    "scipy>=1.7.1, < 1.10.0",
```

File: requirements.txt [hunk 2]
```diff
@@ -3,7 +3,7 @@ bilby>=1.2.0
 bilby_pipe==1.0.5
 numpy>=1.9
 matplotlib>=2.0
-scipy>=1.7.1
+scipy==1.9.3
```
Comment (weizmannk) [inline requirements.txt:6]:
```suggestion
scipy>=1.7.1, < 1.10.0
```

**Review by mcoughlin — APPROVED** [review]
LGTM

**Pull request merged**